.ci/dockerOnLinuxExclusions

```diff
@@ -0,0 +1,13 @@
+# This file specifies the Linux OS versions on which we can't build and
```

Comment:
Couldn't we infer this from the `shouldTestDocker` information? Is there a need to keep this list? Basically, if there isn't a corresponding `:os` project with `shouldTestDocker = true` for the given OS then don't setup the Docker test tasks.

Reply:
We can't. the sub-projects are used only for Vagrant and we need to skip docker on gcp too.  We already ski docker in tests so I don't see the point of `shouldTestDocker`, as we'll always have docker on the host to be able to build the image, and we'll always have to build it for some platform. Not having docker on the vagrant host should error out as it can't be tested at all.

Reply:
Gotcha.

Reply:
We _do_ have  docker on all CI Vagrant hosts. There's no host running `centos-6` we only run that as vagrant guest. And it's ok to auto-detect docker on Vagrant guests, because we have the GCP tests as the source of truth and we are specific about it  there.